components/engine/integration-cli/docker_cli_rmi_test.go

```diff
@@ -150,8 +150,8 @@ func (s *DockerSuite) TestRmiImageIDForceWithRunningContainersAndMultipleTags(c
 	dockerCmd(c, "tag", imgID, newTag)
 	runSleepingContainerInImage(c, imgID)
 
-	out, _, err := dockerCmdWithError("rmi", "-f", imgID)
-	// rmi -f should not delete image with running containers
+	out, _, err := dockerCmdWithError("rmi", imgID)
```

Comment:
This is actually a bug; the change was too wide; see https://github.com/docker/cli/issues/394

(pull request to fix is here: https://github.com/docker/cli/pull/418)